### PR TITLE
mvp2 - synchronize state and blockchain

### PR DESCRIPTION
**Pull request by GusevTimofey**

### CODE REVIEW

File: src/main/scala/mvp2/actors/Accountant.scala [hunk 1]
```diff
@@ -17,18 +18,20 @@ class Accountant extends CommonActor {
...
-        stateRoot = Sha256.toSha256(accountsInfo.toString)
-    }
+    singleParty =>
+      var account: Account = accountsInfo.getOrElse(singleParty._1, Account(singleParty._1, List.empty, 0))
+      singleParty._2.sortBy(_.nonce).foreach { tx =>
+        if (account.nonce + 1 == tx.nonce)
+          account = account.copy(data = account.data :+ tx.data, nonce = tx.nonce)
+      }
+      accountsInfo = accountsInfo + (singleParty._1 -> account)
+      accountsInfo.foreach(x => logger.info(s"${x._1} -> ${stateRoot}"))
```
Comment (ugulavaGeorge) [inline src/main/scala/mvp2/actors/Accountant.scala]: what is the reason?

Comment (GusevTimofey) [reply src/main/scala/mvp2/actors/Accountant.scala]: cntr + alt + l -> code auto clean

**Comment by Bromel777** [comment]
A lot of exceptions during test

**Comment by GusevTimofey** [comment]
> A lot of exceptions during test

it's runtime -> can't create some actors

**Review by ugulavaGeorge — APPROVED** [review]

**Review by Bromel777 — APPROVED** [review]

**Review by kapinuss — APPROVED** [review]

**Pull request closed without merging**